### ISSUE TITLE
Bug Report: 'cutout alpha' has no alpha threshold value to control the cutout edge.

### ISSUE THREAD

**Issue by HogJonny-AMZN** [o3de/o3de#12428]
**Describe the bug**
When 'cutout alpha' is used (in for instance standardpbr), there is no alpha threshold value to control the cutout edge.
This is a standard feature in most game engines with cutout alpha.

**Assets required**
Any standardpbr material that includes an alpha channel texture for cutout.

**Steps to reproduce**
1. Go to the material editor
2. Make astandardPBR material
3. Set the Opacity mode to Cutout, the alpha source to split, and load an alpha map

**Expected behavior**
The Factor slider control is utilized for controlling the alpha threshold, or a threshold slider is added 

**Actual behavior**
There is no slider for controlling the alpha threshold, the factor slider is greyed out.

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/23222931/193937524-ced147e1-1461-4033-a945-848eb83988f8.png)

**Found in Branch**
development

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Win 11
 - CPU Threadripper
 - GPU RTX 3070

**Comment by HogJonny-AMZN** [comment]
![rostlinka_07c_basecolor](https://user-images.githubusercontent.com/23222931/193937902-19a64fef-825c-465a-bd74-1515356695dd.jpeg)
![rostlinka_07c_alpha](https://user-images.githubusercontent.com/23222931/193937903-c129bb96-4c3e-4721-bc92-c9a65a948e18.jpeg)

**Comment by santorac** [comment]
The UV and Factor should not be grayed out, that's a bug, and maybe the main source of the problem you are facing. Here's what I see when opening 009_Opacity_Cutout_SplitAlpha_DoubleSided_LowEndPipeline.material
![image](https://user-images.githubusercontent.com/55155825/195945897-8361eeb4-9275-45e2-a5cc-03e68db9a7f7.png)

I also tried creating a new material as described in the repro steps and could not get into the same scenario. At first they were grayed out, before assigning a texture:
![image](https://user-images.githubusercontent.com/55155825/195946311-49814ff3-bf2f-4a7c-a4a2-382118d47d3a.png)

But as soon as I assigned a texture the fields were enabled.
![image](https://user-images.githubusercontent.com/55155825/195946173-212b7a3f-f179-4606-8647-a9752d506e48.png)

Is there something missing from your repro steps?
